webpack/webpack.dev.js

```diff
@@ -12,6 +29,21 @@ module.exports = merge(common, {
 
   module: {
     rules: [
+      {
+        test: /\.(?:sa|sc|c)ss$/,
```

Comment:
Is this needed for production as well? If it is then it should be in webpack.common.js

Reply:
In product, I intend to use mini-extract-css while using style-loader